biojava-structure/src/main/java/org/biojava/nbio/structure/io/FileConvert.java

```diff
@@ -408,6 +408,10 @@ public static void toPDB(Atom a, StringBuffer str, String chainID) {
 
 
 		Character  altLoc = a.getAltLoc()           ;
+		
+		if ( altLoc == null)
+			altLoc = ' ';
+		
```

Comment:
Good catch @altaite ! It would be better if you commit this bug fix to the master branch directly instead of having it as part of an unrelated pull request. It would also be good to cherry-pick it to the bufixes-4.2 branch so that it goes to next bug fix release. Thanks!